tree/dataframe/inc/ROOT/RSnapshotOptions.hxx

```diff
@@ -29,12 +29,12 @@ struct RSnapshotOptions {
         fSplitLevel(splitLevel), fLazy(lazy)
    {
    }
-   std::string fMode = "RECREATE";             ///< Mode of creation of output file
-   ECAlgo fCompressionAlgorithm = ROOT::kZLIB; ///< Compression algorithm of output file
-   int fCompressionLevel = 4;                  ///< Compression level of output file
-   int fAutoFlush = 0;                         ///< AutoFlush value for output tree
-   int fSplitLevel = 99;                       ///< Split level of output tree
-   bool fLazy = false;                         ///< Delay the snapshot of the dataset
+   std::string fMode = "RECREATE";             //< Mode of creation of output file
+   ECAlgo fCompressionAlgorithm = ROOT::kZLIB; //< Compression algorithm of output file
+   int fCompressionLevel = 1;                  //< Compression level of output file
+   int fAutoFlush = 0;                         //< AutoFlush value for output tree
+   int fSplitLevel = 99;                       //< Split level of output tree
+   bool fLazy = false;                        //< Delay the snapshot of the dataset
```

Comment:
I think doxygen really wants 3 slashes in `///<`?

Reply:
thnx i will fix